docs/rules/index.md

```diff
@@ -35,18 +35,17 @@ Rules in this category are enabled for all presets provided by eslint-plugin-vue
 ## Priority A: Essential (Error Prevention)
 
 - :three: Indicates that the rule is for Vue 3 and is included in all of `"plugin:vue/vue3-essential"`, `"plugin:vue/vue3-strongly-recommended"` and `"plugin:vue/vue3-recommended"` presets.
-- :two: Indicates that the rule is for Vue 2 and is included in all of `"plugin:vue/essential"`, `"plugin:vue/strongly-recommended"` and `"plugin:vue/recommended"` presets.
+- :two: Indicates that the rule is for Vue 2 and is included in `"plugin:vue/essential"` preset.
```

Comment:
All changes in this file are unexpected. Did you forget to run `npm run update` or did the configs actually change? If so, please make sure that the existing configs stay the same.

Reply:
yes, `npm run update` got executed. maybe it was affected by changing the `rule.meta.docs.categories: xxx -> vue2-xxx`. will take a look later.